packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -182,6 +182,10 @@ export const ItemUI = styled.div`
 	width: 100%;
 	color: inherit;
 	opacity: 0.7;
+
+	> svg:first-of-type {
```

Comment:
This selector seems a little bit fragile. Could we add a `className` to the `<Icon>` component or wrap it with a `<span>`?

Reply:
Yeah, this is what I was unsure about. I guess BEM style classnames aren't generally used in components that use css-in-js.

Adding another wrapper isn't great either, but that's what I've gone for here to remove this fragile selector.

If there's another way I'm open to more feedback and can fix it in a follow-up.